### PR TITLE
Change random tips at login to wiki link

### PR DESCRIPTION
**Pull request by TheMitch2**
My intent is to remove the tips.json file and all references to it (no more tips at login) and add a link to the wiki on the "sync complete" dialog.

I spent a fair bit of time trying to track this all down, so if I missed anything please let me know.

### CODE REVIEW

File: src/code/dialogs/authDialog.js [hunk 1]
```diff
@@ -32,16 +32,7 @@ const AuthDialog = WDialog.extend({
     this.randomTip();
```
Comment (cloudkucooland) [inline src/code/dialogs/authDialog.js:32]: you need also to remove this line

**Review by cloudkucooland — CHANGES_REQUESTED** [review]

File: src/code/dialogs/authDialog.js (hunk 1 above)
Comment (TheMitch2) [reply src/code/dialogs/authDialog.js]: Additional commit added.

**Comment by cloudkucooland** [comment]
Travis-CI is saying "  11:14  error  'getLanguage' is defined but never used  no-unused-vars

Look in authDialog and see if getLanguage is imported but never used anywhere, if so, remove the import for it.

**Comment by TheMitch2** [comment]
Link destination changed, build worked as expected. I am happy with the results.

File: src/code/dialogs/authDialog.js [hunk 2]
```diff
@@ -29,19 +29,9 @@ const AuthDialog = WDialog.extend({
...
-    const lang = getLanguage();
-    if (!window.plugin.wasabee.static.tips[lang]) return;
-    const tips = window.plugin.wasabee.static.tips[lang];
-    const keys = Object.keys(tips);
-    // XXX use prompt dialog?
-    alert(tips[keys[(keys.length * Math.random()) << 0]]);
-  },
-
-  _displayDialog: function () {
+    _displayDialog: function () {
```
Comment (le-jeu) [inline src/code/dialogs/authDialog.js:34]:
```suggestion
  _displayDialog: function () {
```

**Review by le-jeu — CHANGES_REQUESTED** [review]

**Comment by TheMitch2** [comment]
Additional le-jeu commits added

**Review by le-jeu — APPROVED** [review]

**Pull request merged**